HelpAssistHer/help-assist-her
# Add Facebook Login Button

## Description
* Add a Facebook login button to the client code
* Get auth to work with Cors
* Add a logout button

## Test Plan
1. Click Login and see that the authentication request was sent
1. Click the "Get One Resource" button and verify it was successful
1. Click Logout and verify the logout request was sent
1. Verify that clicking "Get One Resource" is not successful
